### DISCUSSION TITLE
Add a way to see the activity feed from the online dashboard

### DISCUSSION THREAD

**Discussion by ElektroKill** [community/community#3152] [Mobile]
I would love to see the activity shown on the main page of the desktop website. For examples activities of the people I follow, new stars, commits for watched repositories etc.

**Comment by rnystrom** [comment]
Thanks for the feedback! We have heard this a ton. We're looking into bringing more activity, explore, and trending features to the app in the future. No immediate plans.

If you had to pick one feature from the GitHub.com dashboard feed (seeing stars, commits, follows, etc), which would you want most?

**Reply by ElektroKill** [reply]
I think I would pick commits, it’s very important in my workflow. It’s hard to pick between commits and stars but I think I use commits more!

**Reply by alexisrolland** [reply]
I would love to have the news feed in the mobile app. If I had to pick one feature, I would chose to see the stars. I use it a lot for technology watch. I monitor which repos the people I follow are starting.

Thanks

**Reply by alexisrolland** [reply]
I would love to have the news feed in the mobile app. If I had to pick one feature, I would chose to see the stars. I use it a lot for technology watch. I monitor which repos the people I follow are starring.

Thanks

**Reply by alexisrolland** [reply]
I would love to have the news feed in the mobile app. If I had to pick one feature, I would chose to see the stars. I use it a lot for technology watch. I monitor which repos the people I follow are starring.

Thanks

**Reply by alexisrolland** [reply]
I would love to have the news feed in the mobile app. If I had to pick one feature, I would chose to see the stars. I use it a lot for technology watch. I monitor which repos the people I follow are starring.

Thanks

**Reply by Moc** [reply]
I would definitely like to see updates on issues or discussions as well (newly added & comments on existing)
